src/liblibc/lib.rs

```diff
@@ -1502,8 +1521,9 @@ pub mod types {
                 use types::os::arch::posix88::{dev_t, gid_t};
                 use types::os::arch::posix88::{mode_t, off_t};
                 use types::os::arch::posix88::{uid_t};
+                use types::os::common::posix01::{timespec};
 
```

Comment:
`timespec` is an unused import, and it is an error under `--cfg stage1` build.
`error: unused import, #[deny(unused_imports)] on by default`.